bin/docker-publish.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# TODO: we could also possibly use
+# https://github.com/marketplace/actions/build-and-push-docker-images
+# in place of this.
+
+set -e
+
+# CD over to docker/
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DOCKERDIR="$(dirname "$DIR")/docker"
+cd "$DOCKERDIR" || exit 1
+
+# Extract version, build and push
+SCALAFMT_VERSION=$(git describe --tags | sed 's/^v//')
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
I think it is better to use https://github.com/marketplace/actions/build-and-push-docker-images